README.md

```diff
@@ -1,14 +1,64 @@
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
 # LostAndFound Application
 
 Due to the recent events happening around us. We've decided to develop a full stack web application that helps to report misplaced items in the `EPIC TOWER`.
-The Project makes use of `REACT` framework for it's front-end and `DJANGO` for it's backend.
+
+This repository contains two applications:
+
+1. A [React app](/web-client) that displays information about the missing items or items that have been found by fellows or staffs at Andela.
+See deployed at ...
+2. A [Django server app](/server) that assists the React app by providing data via a REST API.
+See deployed at ...
+
+## React app
+
+Source code located under `/client`. This React app is set up with [Webpack](https://webpack.github.io/) and
+[Express](http://expressjs.com/) so that the React components are rendered in the server side.
+
+#### Getting started
+1. `npm install`: install app dependencies.
+2. Make sure the companion API app is running at`http://localhost:8000/`
+3. `npm start`: start the dev server at `http://localhost:3000/`
+
+#### Deployment
+This app can be deployed on AWS.
+
+## Django server app
+
+Source code located under `/server`. This Django API provides data such as a list of items (both missing and found) and the owners through a REST API. It also provides an admin panel that allows adding, deleting and amending data with ease. It's implemented using [Django REST Framework](http://www.django-rest-framework.org/).
+
+#### Getting started
+* Create a Python virtual environment under `/server`. You can follow [this guide](http://docs.python-guide.org/en/latest/dev/virtualenvs/).
+* Set up the Django settings environment variable: `export DJANGO_SETTINGS_MODULE=lostandfound.settings`. To avoid having to do this every time you can add this line to the `activate` script file located in your virtual environment `env/bin/activate`
+* Activate the environment: `source env/bin/activate`
+* Install dependencies: `pip install -r requirements.txt`
```

Comment:
Since the instruction is to setup the application locally, i believe we should include `python manage.py migrate` after `line 35` before `line 36`, so that the database can be setup locally and the `migrations` we have created can be applied.